adapters/MyTarget/MyTargetAdapter/GADMAdapterMyTargetNative.m

```diff
@@ -148,15 +148,15 @@ - (void)getNativeAdWithAdTypes:(nonnull NSArray<GADAdLoaderAdType> *)adTypes
         (GADNativeAdImageAdLoaderOptions *)adLoaderOptions;
     if (imageOptions.disableImageLoading) {
       _autoLoadImages = NO;
-      cachePolicy = cachePolicy & ~MTRGCachePolicyImages;
+      cachePolicy = MTRGCachePolicyVideo;
```

Comment:
Question: May I ask why the cache policy is `MTRGCachePolicyVideo` if the image loading option is disabled?

Reply:
Default cache policy is ALL (Images + Video), so if Images loading disabled, then Video should stay enabled.